docs/source/guides/monitoring.rst

```diff
@@ -1,12 +1,585 @@
+==========================
 Monitoring with Prometheus
 ==========================
 
+Monitoring stacks usually consist of a metrics collector, a time-series database to store metrics
+and a visualization layer. A popular stack is Prometheus_ with Grafana_ used as the visualization layer
+to create rich dashboards. An architecture of Prometheus and its ecosystem is shown below:
+
+.. image:: ../_static/img/prom-architecture.png
```

Comment:
This graph might be a bit too much for someone just getting started with it - I'd suggest removing it here

Reply:
It's also not particularly related to BentoML, right?

Reply:
yep =( I will remove this